projectq/ops/_metagates_test.py

```diff
@@ -77,7 +77,8 @@ def test_daggered_gate_str():
 
 def test_daggered_gate_tex_str():
```

Comment:
Can you test an additional gate which has a tex_str attribute (e.g. Rx) to cover both branches?